spec/features/request_to_book_spec.rb

```diff
@@ -1,26 +1,31 @@
 feature 'Request space' do
 
+  before :each do
+    host = db_create_user("dude","d@m.com","dude","dude_pass")
+    space_one = db_create_space("first house","pretty",10.01,Date.parse("11 may 2016"),host.id)
+
+    visit '/'
+    click_button('Register')
+    submit_signup_form("guest","guest","g@m.com","guest_pass","guest_pass")
```

Comment:
I think we should be consistent with our quotes: double in HTML and single in Ruby/JS